internal/user/repository/view/model/user.go

```diff
@@ -79,6 +79,7 @@ type HumanView struct {
 	AvatarKey                string         `json:"storeKey" gorm:"column:avatar_key"`
 	Email                    string         `json:"email" gorm:"column:email"`
 	IsEmailVerified          bool           `json:"-" gorm:"column:is_email_verified"`
+	VerifiedEmail            string         `json:"-" gorm:"-"`
```

Comment:
How does this value get scanned when the `gorm:"-"` tag is set?

Reply:
Good question. I tested and it was set, but probably that was a part, which already does not use gorm anymore.
Now added the tag to make sure all parts will be able to handle it.